views/scss/qti/_choice.scss

```diff
@@ -44,6 +44,7 @@
             left: 50%;
             transform: translateX(-50%);
             bottom: calc(100% + 20px);
+            white-space: nowrap;
```

Comment:
This is a backporting for https://github.com/oat-sa/extension-tao-itemqti/pull/2296/files#diff-22f4c217e33cba23248709f515f82a69be5f0d48af8046b39656e938c9729270R47 correct? 

I see the PR I've linked we did not change `views/css/item-creator.css.map`. Is that ok? Is it fixing the missing translations reported by @AntonNeviarouski ?

Reply:
The map file is only there for metadata when debugging, it helps map the code from the bundle to the source code. Being there or not will not impact the feature.